libraries/scripting/dependencies-maven/src/kotlin/script/experimental/dependencies/maven/impl/aether.kt

```diff
@@ -99,7 +102,13 @@ internal class AetherResolveSession(
             WagonTransporterFactory::class.java
         )
 
-        val container = DefaultPlexusContainer()
+        val configuration = DefaultContainerConfiguration().apply {
```

Comment:
Doesn't look relevant to the main commit to me. It will be nice to either explain it or move to a separate commit.

Reply:
Moved to a separate commit